transmart-data/README.md

```diff
@@ -45,7 +46,7 @@ The following are required:
   `PATH` (Oracle and some secondary functionality only)
 * [Kettle][kettle] (ETL only)
 * rsync (Solr only)
-* Java environment (JDK 7)
+* Java environment (JDK 7 or 8)
```

Comment:
Does it work with java 7?

Reply:
I don't know, did not test. My understanding was that it works since it was there before.

Reply:
well, for `transmart-data` java 7 would work, yes. I remember having problems to run the web app though.